src/lib/utils/jest/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { render } from '@testing-library/react';
+
+import { InstantSearch } from 'react-instantsearch-core';
+import { BrowserRouter } from 'react-router-dom';
+
+jest.mock('algoliasearch');
```

Comment:
Was this a pain to get testing working? I coded this part in a hurry so I wouldn't be surprised if it was. On the cool note they have hooks now I think.

Reply:
The only pain was figuring out components that have the search bar need a special component wrapper to work, which is why I added this